aria-practices.html

```diff
@@ -3697,6 +3697,1040 @@ <h5>Examples</h5>
+            <li>If an element is referenced by <code>aria-labelledby</code> more than one time during a name calculation, the second and any subsequent references will be ignored.</li>
+            <li>There are certain types of elements, such as paragraphs and list items,  that should not be named with <code>aria-labelledby</code>. They are identified in the table in the <a href="#naming_role_guidance">Accessible Name Guidance by Role</a> section.</li>
+            <li>
+          If <code>aria-labelledby</code> is applied to an element with one of the roles that supports <a href="#naming_with_child_content">naming from child content</a>, content contained in the element and its descendants is hidden from assistive technology users unless it is also referenced by <code>aria-labelledby</code>.
+          It is strongly recommended to avoid using this attribute to override content of one of these elements except in rare circumstances where hiding content from assistive technology users is beneficial.
+        </li>
+            <li>Because calculating the name of an element with <code>aria-labelledby</code> can be complex and reference hidden content, testing with assistive technologies to ensure the expected name is presented to users is particularly important.</li>
+          </ol>
+        </section>
+
+        <section id="naming_with_labels">
+          <h5>Naming Form Controls with the Label Element</h5>
+          <p>
+            The HTML <code>label</code> element enables authors to identify content that serves as a label and associate it with a form control.
+            When a <code>label</code> element is associated with a form control, Browsers calculate an accessible name for the form control from the <code>label</code> content.
+          </p>
+          <p>
+            For example, text displayed adjacent to a checkbox may be visually associated with the checkbox, so it is understood as the checkbox label by users who can perceive that visual association.
+            However, unless the text is programmatically associated with the checkbox, assistive technology users will experience a checkbox without a label.
+            Wrapping the checkbox and the labeling text in a <code>label</code> element as follows gives the checkbox an accessible name.
+          </p>
+          <pre><code>
+  &lt;label>
+    &lt;input type="checkbox" name="subscribe">
+    subscribe to our newsletter
+  &lt;/label>
+          </code></pre>
+          <p>
+            A form control can also be associated with a label by using the <code>for</code> attribute on the <code>label</code> element.
+            This allows the label and the form control to have different parents in the DOM, but requires adding an <code>id</code> attribute to the form control, which can be error-prone.
```

Comment:
"This allows the label and the form control to have different parents in the DOM, "

... true, but it also allows the label and the form control to be siblings. So the above phrase may be misleading as-is.